### ISSUE TITLE
Switch Mathjax JS package source

### ISSUE THREAD

**Issue by takluyver** [jupyter/notebook#3755]
From #3751; cc @minrk @gnestor.

Should we use https://github.com/mathjax/MathJax in bower? This saves Min maintaining the `components/Mathjax` repository if it works.

Should we move away from Bower entirely? It seems to be no longer the in thing. We tried to move away from it before, and ran into problems, but Kyle mentioned that we were trying to switch away from requirejs at the same time, and that was actually the main issue. I'm not sure which other package manager we'd switch to, though. npm?

**Comment by gnestor** [comment]
I have tested using `^2.7.4` vs. `components/MathJax#~2.7.4` and the results look identical to me. 

As for migrating from bower to npm, it would be nice to switch to npm as that is the current trend in the JS world, but bower works and doesn't require and transpiling (npm packages targeting node need to be transpiled using browerify, webpack, etc.)

**Comment by minrk** [comment]
Yeah, switching away from require was the source of the problem, I don't think switching away from bower should be an issue as long as everything is on npm, which is probably is.
